src/main/java/com/google/api/codegen/discovery/config/SampleConfig.java

```diff
@@ -88,6 +88,17 @@
   public abstract String apiTypeName();
 
   /**
+   * Returns the lower-camel formatted API type name.
+   *
+   * This is mostly for utility for overrides where apiTypeName is used to
```

Comment:
Fixed by deletion - Garrett suggested a much cleaner fix to this problem that doesn't require a function in `SampleConfig`